modules/quick-start-user-workflow.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * web_console/creating-quick-start-tutorials.adoc
+
+[id="quick-start-user-workflow_{context}"]
+= Quick start user workflow
+
+. In the *Administrator* or *Developer* perspective, click the *Help icon* and select *Quick starts*.
```

Comment:
Currently in 4.7, it is 'Quick Starts'
<img width="255" alt="Screen Shot 2021-02-10 at 10 50 49 AM" src="https://user-images.githubusercontent.com/12692381/107457927-faf73d80-6b8d-11eb-99dc-0b063a02dda1.png">